src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -1347,11 +1348,11 @@ RequestStatus GNAPlugin::WaitFor(uint32_t request_idx, int64_t millisTimeout) {
         ExportScores(output_blob->buffer(),
                      gna_output_blob->cbuffer(),
                      gna_output_desc.orientation,
-                     batchSize,
-                     batchSize,
-                     elementsPerBatch,
-                     elementsPerBatch,
-                     elementsPerBatch,
+                     static_cast<uint32_t>(batchSize),
```

Comment:
what is a type of batchSize and elementsPerBatch?
cannot they be declared as uint32_t to avoid casting in many places?